src/List/organisation-unit-dialog/OrgUnitDialog.component.js

```diff
@@ -144,6 +144,16 @@ class OrgUnitDialog extends React.Component {
                     roots={this.props.roots}
                     selected={this.state.selected}
                     intersectionPolicy={true}
+                    filteringByNameLabel={
+                        title.includes("organisation units capture")
```

Comment:
This kind of checks are very brittle, whenever we change the literal it will stop working. I'd add these two props to the component and the caller must pass them. 

[minor] Also, not sure about passing the translation key as prop. Maybe just the already translated string?